test/e2e/tests/004-surveyFormEditCluster.js

```diff
@@ -0,0 +1,81 @@
+import { click, expectExists, getElement, clearTextBox, toRightOf, writeIntoTextBox } from '../utils/api'
+import {
+  clickSidebarBtnSurveyForm,
+  waitForLoader,
+  addItemToPage,
+  expectItemIsTheLastNodeDef,
+  expectItemsAreInOrderAsNodeDef,
+} from '../utils/ui'
+
+const expectSurveyFormLoaded = async () => expectExists({ selector: '.survey-form' })
+
+const expectHasOnlyRootEntity = async ({ rootEntityName }) => {
+  await expectExists({ selector: '.btn-node-def', numberOfItems: 1 })
+  await expectExists({ selector: `//button[text()='${rootEntityName}']`, numberOfItems: 1 })
```

Comment:
isn't numberOfItems by default 1? you can omit it here

Reply:
I put one because exists is not the same as being an exact number of times for example  here root_entity is two times:

![Captura de pantalla 2020-09-03 a las 11 21 01](https://user-images.githubusercontent.com/11272068/92096764-8d253980-edd7-11ea-8ba2-95b0c158fc19.png)

If we like that something is on the screen it should exist and if we like to be sure that something is exactly one time on the screen we should check numberOfItems: 1.

What do you thing?

Reply:
I see now.
then I think we should have two methods.
one is expectToBeTruthy and another one expectToBe.
what do you think?

Reply:
How about
expectToBeTruthy -> is **expectExists**
expectToBe -> **expectToBe** ({ numberOfTimes })

Reply:
can I do the Refactor into the next PR 005 because I am going to have a lot of conflicts?

Reply:
sure